llvm/lib/Transforms/InstCombine/InstCombineCalls.cpp

```diff
@@ -1174,6 +1174,86 @@ static Instruction *moveAddAfterMinMax(IntrinsicInst *II,
   return IsSigned ? BinaryOperator::CreateNSWAdd(NewMinMax, Add->getOperand(1))
                   : BinaryOperator::CreateNUWAdd(NewMinMax, Add->getOperand(1));
 }
+
+// Try canonicalize max(max(X,C1) binop C2, C3) -> max(X binop C2, C3)
```

Comment:
```suggestion
/// Try canonicalize max(max(X,C1) binop C2, C3) -> max(X binop C2, C4)
```
Use `///` for header comments.